resources/documentation/multiple_icebergs.rst

```diff
@@ -227,45 +291,226 @@ And add management2 to iceberg2, by editing /etc/bluebanquise/inventory/cluster/
-  management2
+  mngt2
+
+Play again playbooks on mngt1, so mngt2 is added into dhcp, pxe, dns, hosts,
+etc. configuration files.
+
+.. note::
+  Even if mngt2 is not part of iceberg1, it has been added to configuration
+  files on mngt1, like any other nodes of iceberg1. All nodes part of a sub
+  iceberg and also part of *mg_managements* group are automatically added, as
+  they also need to be deployed from this iceberg, like any other nodes.
+
+Once done, use standard procedure to deploy OS on mngt2 from mngt1 (*bootset*,
+etc).
+
+Now, few steps has to be followed in a specific order in order to deploy
+configuration on mngt2.
+
+Deploy sub management configuration
+-----------------------------------
+
+First, using default strategy (you can use another one), it is needed that mngt2
+mount over nfs the repositories and the BlueBanquise configuration from mngt1.
+This to be able to install packages, but also act as a repository server for
+its iceberg, and be able to deploy the configuration on its iceberg nodes.
+
+We need to ensure mngt2 is part of a group that will mount the repositories and
+bluebanquise, in nfs.yml. By default, this group is called
+*secondary_managements*.
+
+Create file *inventory/cluster/groups/secondary_managements* with the following
+content:
+
+.. code-block:: text
+
+  [secondary_managements]
+  mngt2
+
+Then ensure in file *inventory/group_vars/all/general_settings/nfs.yml* you have
+exactly at least these two exports:
+
+.. code-block:: yaml
+
+  nfs:
+
+    ...
+
+    bluebanquise:
+      mount: /etc/bluebanquise
+      export: /etc/bluebanquise
+      server: mngt1
+      clients_groups:
+        - secondary_managements
+      take_over_network: ice1-1
+      export_arguments: ro,no_root_squash,sync
+      mount_arguments: ro,intr,nfsvers=4.2,bg
+
+    repositories:
+      mount: /var/www/html/repositories
+      export: /var/www/html/repositories
+      server: mngt1
+      clients_groups:
+        - secondary_managements
+      take_over_network: ice1-1
+      export_arguments: ro,no_root_squash,sync
+      mount_arguments: ro,intr,rsize=32768,wsize=32768,nfsvers=4.2,bg
+
+So mngt1 will export both folders, and members of secondary_managements (so
+mngt2) will mount it.
+
+.. note::
+  All is set to read only here (ro). It is up to you to switch to read write if
+  needed.
 
-Play again playbooks on management1, so management2 is added into dhcp, pxe, dns, hosts, etc. configuration files.
+If you just added these new nfs exports, play the role nfs_server on mngt1 and
+check that mngt1 now export these foldes, using *showmount -e mngt1* command.
 
-Once done, use standard pxe boot procedure to deploy OS on management2.
+Next, we will need a playbook for mngt2. Copy current mngt1 dedicated playbook:
 
-Now, few steps has to be followed in a specific order in order to deploy configuration on it.
+.. code-block:: text
+
+  cp /etc/bluebanquise/playbooks/mngt1.yml /etc/bluebanquise/playbooks/mngt2.yml
```

Comment:
What is the source of /etc/bluebanquise/playbooks/mngt1.yml? Did we create it earlier in the procedure?

Reply:
It was supposed created previously by admin during pre-multiiceberg phase. Maybe not clear.

Reply:
This is the only place I can find this file. It is obvious to me, but other people may struggle here. Not a blocker for the merge anyway.
```
$ grep -r mngt1.yml resources/documentation/
resources/documentation/multiple_icebergs.rst:  cp /etc/bluebanquise/playbooks/mngt1.yml /etc/bluebanquise/playbooks/mngt2.yml
```

Reply:
You are right. I will leave a comment just above it.

Reply:
Added a note